packages/ui/src/council/modals/VoteForCouncil/VoteForCouncilSuccessModal.tsx

```diff
@@ -30,9 +31,10 @@ export const VoteForCouncilSuccessModal = ({ onClose, candidateId }: Props) => {
 
       <SuccessModalBody>
         <TextMedium margin="l" light>
-          You have just successfully voted for the Candidate
+          You have just successfully voted for the Candidate. This is a good time to backup your votes.
         </TextMedium>
-
+        <BackupVotesButton cycleId={candidate?.cycleId} />
+        ￼
```

Comment:
@traumschule is this on purpose ?

Reply:
i added empty lines around `BackupVotesButton` for clarify but prettier had another view, or do you mean something else?